src/components/Modal/components/SbModalHeader.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div
+    v-if="title"
+    class="sb-modal-header"
+    :class="{ 'sb-modal-header--left': returnAlignment }"
```

Comment:
Will this always be left aligned?

Reply:
The modal header can be rendered in both the `center` and the `left`, the computed `returnAlignment` returns **true** for when the alignment is on the left, as the default is in the `center`.